libs/gltfio/src/FilamentAsset.cpp

```diff
@@ -81,7 +81,7 @@ FFilamentAsset::~FFilamentAsset() {
         mEngine->destroy(ib);
     }
     for (auto tx : mTextures) {
-        if (UTILS_LIKELY(tx.isOwner)) {
+        if (UTILS_LIKELY(tx.isOwner) && tx.texture != nullptr) {
```

Comment:
This check is not necessary.

Reply:
yes  not necessary because `Engine::destroy` handles `nullptr`.